staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1186,6 +1186,83 @@ message ContainerConfig {
 
     // CDI devices for the container.
     repeated CDIDevice CDI_devices = 17;
+
+    // Lifecycle includes the custom stop signal for the container
+    Lifecycle lifecycle = 18;
+}
+
+message Lifecycle {
+    // The custom stop signal for the container
+    Signal stop_signal = 1;
+}
+
+enum Signal {
+    RUNTIME_DEFAULT   = 0;
```

Comment:
Alignment is off.

Reply:
Please check https://github.com/kubernetes/kubernetes/pull/130556/commits/c8a8b89ef6643dee782d7fedf09cfef5ec41568b.